docs/docsite/rst/dev_guide/developing_collections_shared.rst

```diff
@@ -65,6 +65,21 @@ By wrapping the import statement in a ``try`` block, the payload builder will om
 
 The optional import behavior also applies to module_utils imported from collections.
 
+.. _collection_dependencies:
+
+Adding collection dependencies
+==============================
+
+If your collection depends on features and functionality from another collection, ????
```

Comment:
Incomplete sentence and question marks at the end?

Reply:
Phew, thanks for catching that @sagidlow. I need to finish out this introductory statement for collection dependencies.

Reply:
Fixed in 2b84951fc61422551f159ccf5df686beff5fb23e.